src/components/Pages/ManageWiki/Cards/Skin.vue

```diff
@@ -22,45 +20,62 @@
         <span>It may take up to 10 seconds for changes to be reflected on your wiki</span>
       </v-tooltip>
     </v-card-actions>
+    <Message ref="message" />
   </v-card>
 </template>
 
 <script>
+import Message from '../Features/Message.vue'
+
 export default {
   name: 'Skin',
+  components: {
+    Message
+  },
   props: [
     'wikiId'
   ],
   data () {
     return {
-      items: [
-        'Vector',
-        'Modern',
-        'Timeless'
+      skins: [
```

Comment:
I would expect `minerva` and `vector-2022`  to be part of this array. As I understood we want them added to the skins

Reply:
ah right, so we removed them again because they are not ready to be used yet. After our next mediawiki update we will be able to include them here. Until then I changed this PR so that it's just some code refactoring, basically preparation to add them in the future + reuse of the snackbar for alerts

Reply:
I see I will then merge this